open-metadata-implementation/access-services/subject-area/subject-area-server/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/handlers/SubjectAreaHandler.java

```diff
@@ -193,7 +253,11 @@ CategorySummary getCategorySummary(String restAPIName,
                                            String entityGuid,
                                            FindRequest findRequest) throws SubjectAreaCheckedException,
                                                                            PropertyServerException,
-                                                                           UserNotAuthorizedException {
+                                                                           UserNotAuthorizedException, org.odpi.openmetadata.commonservices.ffdc.exceptions.InvalidParameterException {
+        if (findRequest.getPageSize() == null) {
```

Comment:
Please remove org.odpi.openmetadata.commonservices.ffdc.exceptions. from exception

Reply:
So the  invalidParameterHandler.validatePaging is a standard handler that is expected to be used to police paging parameters across Egeria OMASes, it can be called client side and server side. It throws this exception - I do not think it is worth changing it to a subjectArea Exception as it never leaves the method.